cypress/integration/MultiSelect/position/index.js

```diff
@@ -29,11 +30,34 @@ When('the window is scrolled down', () => {
     cy.scrollTo(0, 800)
 })
 
+When('the window is resized to a greater width', () => {
+    cy.viewport(1200, 660)
+    // Wait until the DOM is painted
+    cy.wait(100)
```

Comment:
A similar/identical issue as the one mentioned in this https://github.com/dhis2/ui/pull/10#discussion_r408376841. In both cases the reference element changes size and the ResizeObserver picks up on these changes to update the popper element. It seems cypress asserts just before the ResizeObserver kicks into action.....